### ISSUE TITLE
Crowd sources gear inventory

### ISSUE THREAD

**Issue by possibly** [galenmaly/lighterpack#34]
It'd be great to have some sort of way to add gear that other people have already filled out the info for. This would make building a new list of gear really fast since one could just search for their gear and add it right into their list, complete with a picture, weight, etc.

**Comment by johnabela** [comment]
I tried doing this on my gear list service (no longer online) that had a couple hundred hikers, and it just turned into an utter nightmare.

Not only were people adding duplicate companies _(ie: zpacks, z packs, z-packs, and every combination thereof)_ but people were adding the same product over and over and over, all because they were a gram or two heavier than the other five bazillion entries for the same product. Not to mention colour variants, or size variants, or hip belt variants, or zipper or velcro variants, or variants this or that.

It was probably the number one reason I pulled my gear list service offline - trying to keep the sanity of all that to a minimum just got to be too much.

**Comment by jimjkelly** [comment]
Perhaps just a set of common basics?  Definitely would be helpful to have something to start with and not have to weigh everything.

**Comment by johnabela** [comment]
@jimjkelly -- and, what happens when a company releases a product during the next production cycle, continues to use the same product name and SKU, yet it has a different weight. Somebody is going to have their weights skewed, or, there are going to be multiple entries for the same product/sku, at which you are _still_ going to have to weigh the one you got to see which one one of the potential dozens in the list matches.

**Comment by jimjkelly** [comment]
From my perspective I see the value in something like this as being able to do quick and dirty estimation without having to bother weighing everything individually, so your scenario isn't all that concerning.  At the point at which I'm bothering to weigh things myself the tool begins to lose its usefulness and I might as well have it in a text document or spreadsheet.

I understand that might not be true for others, and that's absolutely fine.  Not here to say anything *must* be how I'm saying.  Simply confirming OP's interest in a feature, and stating why.

**Comment by johnabela** [comment]
Yep yep!  I think if we kept weight out of it, and just had a .json file with a list of manufacturers and top selling products, that could at least be the basis for where to take all of this.
